draft-ietf-httpbis-client-hints.md

```diff
@@ -190,6 +200,12 @@ Implementers ought to consider both user and server controlled mechanisms and po
 Implementers SHOULD support Client Hints opt-in mechanisms and MUST clear persisted opt-in preferences when any one of site data, browsing history, browsing cache, or similar, are cleared.
 
 
+## Abuse Detection
+A user agent that tracks access to active fingerprinting information SHOULD consider emission of Client Hints headers similarly to the way it would consider access to the equivalent API.
+
+User agents which detect abuse through forensic means can observe HTTP responses to requests containing Client Hints, and their variance based on those hints. As Client Hints are provided in order to allow variation and optimization of the content to the provided Client-Hints, resource responses that do not vary on the Client Hints provided can indicate that such Hints are being used for nefarious purposes. Detection of variance can be done using the explicit Vary header, or by observing content changes as a result of different Client Hints sent.
```

Comment:
"their" variance?

The problem here is that it is difficult to identify variance from a single instance.  You need to establish patterns of behaviour, which means comparing the response to a baseline.  Maybe "Research into abuse of Client Hints might look at how HTTP responses that contain Client Hints differ from those without.  This might be used to reveal which Client Hints are in use, allowing researchers to further analyze that use."

I don't think that you need the mention of Vary here.  Bad actors won't use that, or might use `Cache-Control: no-store` and friends.

Reply:
Cool, incorporated your language (while mentioning different values, on top of a with/without comparison)